package.json

```diff
@@ -37,7 +37,8 @@
     "clean": "git clean -fX ./dist ./*.xpi",
```

Comment:
Does this also need to be updated to cleanup `./build` instead of `./*.xpi`?

Reply:
Yup. 

Is it worth taking out the `./*.xpi` both from here and `.gitignore` since the packages won't be at top-level as soon as this is merged?

Reply:
> Is it worth taking out the ./*.xpi both from here and .gitignore since the packages won't be at top-level as soon as this is merged?

No, it's not.  ♻️